app/uk.gov.hmrc.helptosavestridefrontend/controllers/StrideController.scala

```diff
@@ -252,30 +264,30 @@ class StrideController @Inject() (val authConnector:       AuthConnector,
 
   def getAccountCreatedPage: Action[AnyContent] = authorisedFromStride { implicit request ⇒
     checkSession(SeeOther(routes.StrideController.getEligibilityPage().url),
-                 whenEligible   = { (_, detailsConfirmed, nsiUserInfo) ⇒
+                 whenEligible   = { (_, detailsConfirmed, nsiUserInfo, mayBeAccountNumber) ⇒
         if (!detailsConfirmed) {
           SeeOther(routes.StrideController.customerEligible().url)
         } else {
-          sessionStore.store(HtsSession(SessionEligibilityCheckResult.AlreadyHasAccount, nsiUserInfo)).fold({
+          sessionStore.store(HtsSession(SessionEligibilityCheckResult.AlreadyHasAccount, nsiUserInfo, detailsConfirmed, mayBeAccountNumber)).fold({
             e ⇒
-              logger.warn(s"Could not write session to mongo dueto: $e")
+              logger.warn(s"Could not write session to mongo due to: $e")
               SeeOther(routes.StrideController.getErrorPage().url)
           }, { _ ⇒
-            Ok(views.html.account_created())
+            Ok(views.html.account_created(mayBeAccountNumber))
```

Comment:
at this point, there's no reason for the account number being passed in to be optional - we could just pass in a defined account number

Reply:
fixed